core/tests/protractor/learnerDashboard.js

```diff
@@ -39,210 +42,309 @@ var SubscriptionDashboardPage =
-    element.all(by.css(
-      '.protractor-test-collection-summary-tile-title')).first().click();
-    // Go to the first and only exploration.
-    element.all(by.css(
-      '.protractor-test-collection-exploration')).first().click();
-    // Leave the exploration inbetween. The collection should be found in the
-    // 'In Progress' section.
-    explorationPlayerPage.submitAnswer('Continue', null);
-    browser.ignoreSynchronization = true;
-    learnerDashboardPage.get();
-    general.acceptAlert();
-    browser.waitForAngular();
+    // Create new 'Test Collection' containing exploration 'About Oppia'.
+    users.login('testCollectionAdm@learnerDashboard.com', true);
+    creatorDashboardPage.get();
+    creatorDashboardPage.clickCreateActivityButton();
+    creatorDashboardPage.clickCreateCollectionButton();
+    collectionEditorPage.searchForAndAddExistingExploration(
+      'Head of Collection');
+    collectionEditorPage.saveDraft();
+    collectionEditorPage.closeSaveModal();
+    collectionEditorPage.publishCollection();
+    collectionEditorPage.setTitle('Test Collection');
+    collectionEditorPage.setObjective('This is a test collection.');
+    collectionEditorPage.setCategory('Algebra');
+    collectionEditorPage.saveChanges();
     general.waitForSystem();
-    browser.ignoreSynchronization = false;
-    learnerDashboardPage.navigateToIncompleteCollectionsSection();
-    learnerDashboardPage.expectTitleOfSummaryTileToMatch('Test Collection');
+    users.logout();
 
-    // Go to the test collection.
-    browser.get('/search/find?q=');
-    browser.waitForAngular();
-    general.waitForSystem();
-    element.all(by.css(
-      '.protractor-test-collection-summary-tile-title')).first().click();
+    users.login('learner4@learnerDashboard.com');
+    // Go to 'Test Collection' and play it.
+    libraryPage.get();
     general.waitForSystem();
-    // Go to the first and only exploration.
-    element.all(by.css(
-      '.protractor-test-collection-exploration')).first().click();
-    // Complete the exploration. The collection should be found in the
-    // 'Completed' section as the collection is also completed.
-    explorationPlayerPage.submitAnswer('Continue', null);
-    explorationPlayerPage.submitAnswer(
-      'MultipleChoiceInput', 'Those were all the questions I had!');
-    explorationPlayerPage.submitAnswer('Continue', null);
+    libraryPage.findExploration('Test Collection');
+    libraryPage.playCollection('Test Collection');
+    var firstExploration = element.all(
+      by.css('.protractor-test-collection-exploration')).first();
+    // Click first exploration in collection.
+    browser.wait(until.elementToBeClickable(firstExploration), 10000,
+      'Could not click first exploration in collection')
+      .then(function(isClickable) {
+        if (isClickable) {
+          firstExploration.click();
+        }
+      });
+
+    // TODO(hoangviet1993): Leave the collection in between.
```

Comment:
Seems to be a repeat of above comment; same question applies here too.

Reply:
I have completed this TODO :+1: